examples/model_hosting/data_fetcher/data_spec_schedule.py

```diff
@@ -26,7 +26,7 @@
             "gas_gain": ScheduleInputTimeSeriesSpec(3658191334677419),
         }
     ),
-    output=ScheduleOutputSpec({"transformed": ScheduleOutputTimeSeriesSpec(123)}),
+    output=ScheduleOutputSpec({"transformed": ScheduleOutputTimeSeriesSpec(id_of_output_timeseries)}),
```

Comment:
Also I didn't get any output from `gas_gain`, `gas_integ_time` when I ran this. Could you try running this example again?